### ISSUE TITLE
Organização do projeto

### ISSUE THREAD

**Issue by arthurTemporim** [lappis-unb/Bottis#13]
Pastas como:
* `bot`
* `coach`
* `docs`
* `notebooks`
* `elasticsearch`
* `scripts`

estão presentes no projeto, porém acho que apenas as pastas:

* `docker`
* `rabbitmq`
* `policies`
* `union_server`

são necessárias para o projeto.

@pablodiegoss @MatheusMiranda Vocês podem confirmar isso?

Se sim precisamos atualizar a organização do projeto, de forma a ficar mais enxuto.

**Comment by pablodiegoss** [comment]
Concordo, mas preciso verificar a necessidade de ter uma pasta bot, já que ainda não integramos a versão 1 do rasa no bottis e também temos a inserção "obrigatória" da policy e action no bot pra podermos executar a confraria
